src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,14 +1,44 @@
-import React from 'react';
+import { useDispatch } from 'react-redux';
+import { useAppSelector } from '../../app/hooks';
+import { actions as filterActions } from '../../features/filter';
+
+import { StatusTodos } from '../../types/StatusTodos';
+import { ShowTodos } from '../../types/ShowTodos';
 
 export const TodoFilter: React.FC = () => {
+  const dispatch = useDispatch();
+  const query = useAppSelector(state => state.filter.query);
+
+  const handleFilterStatus = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    switch (event.target.value) {
+      case StatusTodos.Active:
+        dispatch(filterActions.setStatus(ShowTodos.Active));
```

Comment:
You can destructure the action from the filterActions for better readability

Reply:
Done